profiles/kartikver15gr8.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/91731654/188267011-44c4fd62-260f-4b8c-b413-ed48916176d0.png",
+    "issueId": 1355,
+    "name": "Kartikey Verma",
+    "username": "kartikver15gr8"
+  }
```

Comment:
Please remove the extra space before the closing bracket
```diff
-  }
+ }
```

Reply:
Sure